barman/image-pre-start.d/01_update_postgresql.conf.sh

```diff
@@ -3,16 +3,22 @@
 set -eo pipefail
 
 . /usr/local/bin/functions.sh
+major_gte() { verlt $1 ${PG_MAJOR}; }
```

Comment:
Please move this into functions.sh to keep it with all of the other version comparison functions.